srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -340,6 +337,42 @@ private void retrieveAdlNotes(final int dfn, final Patient patient)
         }
     }
     
+    private void retrieveDnrNotes(final int dfn, final Patient patient)
+    {
+        try
+        {
+            final List<String> rpcResults = fProcedureCaller.doRpc(
+                    fDuz,
+                    RemoteProcedure.GET_NOTES_WITH_SUBSTRING,
+                    String.valueOf(dfn),
+                    "DNR");
+            // If the resultString is a success, add it to the patient's dnr notes.
+            // Else, we don't need to do anything.
+            if(!rpcResults.isEmpty())
+            {
+                final ReferenceNotes dnrNotes = getReferenceNotes(rpcResults);
+                patient.getDnrNotes().clear();
+                patient.getDnrNotes().addAll(dnrNotes.getAllNotes());
+            }
+        }
+        catch(final Exception e)
+        {
+            // If an exception occurs for any reason, log a warning but allow the application
+            // to continue without failure.
+            LOGGER.warn("Unable to retrieve patient's DNR notes. {}", e);
+        }
+    }
+    
+    private ReferenceNotes getReferenceNotes(final List<String> rpcResults) throws JAXBException
```

Comment:
I think this method could use a doc comment, particularly why it takes a `List<String>` instead of a String.